### PR TITLE
[MultiDB] should use multiDB API

### PR DESCRIPTION
**Pull request by dzhangalibaba**
* Should use new multiDB Connector in daemon base
* This submodule needs updated with daemon_base changes together which is in sonic_buildinage
 

Signed-off-by: Dong Zhang d.zhang@alibaba-inc.com

### CODE REVIEW

**Comment by dzhangalibaba** [comment]
This PR and https://github.com/Azure/sonic-buildimage/pull/4549 depends on each other.
the submodule needs updated with the merge of this PR together.

**Comment by dzhangalibaba** [comment]
@qiluo-msft

File: sonic-ledd/scripts/ledd [hunk 1]
```diff
@@ -76,7 +76,7 @@ class DaemonLedd(DaemonBase):
             sys.exit(LEDUTIL_LOAD_ERROR)
 
         # Open a handle to the Application database
-        appl_db = daemon_base.db_connect(swsscommon.APPL_DB)
```
Comment (lguohan) [inline sonic-ledd/scripts/ledd:79]: why not use swsscommon.APPL_DB?

Comment (dzhangalibaba) [reply sonic-ledd/scripts/ledd:79]: swsscommon.APPL_DB is db id from swsscommon c++ library. Here we need a string "APPL_DB", For backward compatibility, we leave db id still there and not changed APPL_DB = "APPL_DB"

**Comment by dzhangalibaba** [comment]
@qiluo-msft  can we merge this first? then I'll update another PR "[MultiDB] daemon base should use multiDB DBConnector #4549"  with this submodule update as well to avoid the dependency.

**Pull request merged**